client/src/components/payment/payment-modal.vue

```diff
@@ -59,6 +59,7 @@ import PaymentSessionMixin from '@/components/payment/payment-session-mixin'
 import NameRequestMixin from '@/components/mixins/name-request-mixin'
 import DisplayedComponentMixin from '@/components/mixins/displayed-component-mixin'
 import { getBaseUrl } from './payment-utils'
+import newReqModule from '@/store/new-request-module'
```

Comment:
I believe this is not needed, since `this.redirectToPaymentPortal` should already be imported from PaymentMixin (line 57 above).

Reply:
(Doesn't hurt though, so this is not blocking my approval of this PR.)